doc/modules/cross_validation.rst

```diff
@@ -209,6 +209,19 @@ The following sections list utilities to generate indices
 that can be used to generate dataset splits according to different cross
 validation strategies.
 
+Cross-validation iterators for i.i.d. data.
+===========================================
+
+Data is Independent Identically Distributed (i.i.d.) if the individual samples
+of the data are independent of each other and are drawn from an identical
+distribution.
+
+Simply put, it means that each data point is created equally, and there is no
+covariance between the different data points.
```

Comment:
I find this is confusing. Maybe with the following would be better:

Simply put, samples are assumed to be created from the same generative process and
that generative process is assumed to have no memory of past generated samples.